frontend/src/components/Profile/ProfilePreview/ProfileMeta/ProfileCity/ProfileCity.module.scss

```diff
@@ -0,0 +1,50 @@
+@import "utils";
+
+.profileCity {
+  display: flex;
+  gap: 4px;
+
+  @include icon-styles {
+    width: 24px;
+    height: 24px;
+    margin-right: 0;
+  }
+
+  &--linked {
```

Comment:
are dashes available while using in the component styles?
also, as we're using CSS modules, it's not obligatory to use BEM

Reply:
> ivadimko

there are some tricks on how to use them (shown at screenshot, `linkedProfileClass` variable)
<img width="873" alt="image" src="https://github.com/mate-academy/huntd-public/assets/129086393/9e212693-1893-4425-a21c-69d24abcb3b5">

but I got your idea and I'll change it.